Meta/Lagom/CMakeLists.txt

```diff
@@ -24,16 +24,43 @@ get_filename_component(
     ABSOLUTE CACHE
 )
 
+list(APPEND CMAKE_MODULE_PATH "${SERENITY_PROJECT_ROOT}/Meta/CMake")
+
+if(NOT COMMAND serenity_option)
+    macro(serenity_option)
+        set(${ARGV})
+    endmacro()
+endif()
+
+include(lagom_options)
+
+if (ENABLE_ALL_THE_DEBUG_MACROS)
+    include(all_the_debug_macros)
+endif(ENABLE_ALL_THE_DEBUG_MACROS)
```

Comment:
Oh no, let's not repeat the condition in the ending command.

Reply:
Oh, true. not sure how that syntax got like that. probably copy  pasted from the main cmake lists and fixed the style in the original. fixed in ea349c1e54a2d17a84219347401eef99cf050790